python_modules/libraries/dagster-databricks/dagster_databricks/ops.py

```diff
@@ -25,6 +25,7 @@ def create_databricks_run_now_op(
     poll_interval_seconds: float = DEFAULT_POLL_INTERVAL_SECONDS,
     max_wait_time_seconds: float = DEFAULT_MAX_WAIT_TIME_SECONDS,
     name: Optional[str] = None,
+    databricks_resource_key: Optional[str] = "databricks",
```

Comment:
This type should just be `str`, not `Optional[str]`, as `None` is not a valid key to the required_resource_keys dictionary.